src/app/containers/MostRead/index.jsx

```diff
@@ -46,14 +46,14 @@ const MostReadContainer = ({
 MostReadContainer.propTypes = {
   mostReadEndpointOverride: string,
   constrainMaxWidth: bool,
-  maxTwoColumns: bool,
+  columnLayout: oneOf(['oneColumn', 'twoColumn', 'multiColumn']),
```

Comment:
Same as above.
https://github.com/bbc/simorgh/pull/5739/files#diff-04298a8f3ec1cbafcb68a13991abd880R196

Reply:
@EinsteinNjoroge Wouldn't we still need to declare the props as they still need to be passed down to the canonical component?

Reply:
Sure, this makes sense. We can leave it as is.